modules/home-environment.nix

```diff
@@ -291,6 +291,15 @@ in
       '';
     };
 
+    home.sessionVariablesInit = mkOption {
```

Comment:
I'd make this `internal`, otherwise it's not clear whether to use this or `sessionVariablesExtra`.

Reply:
Yeah I was thinking maybe you might want to override it for whatever reason, but probably not.

If it's going to be internal then another alternative would be to expose the derivation that creates `<profile>/etc/profile.d/hm-session-vars.sh` rather than the text, which might generalize more elegantly if there's more things to translate in the future (e.g. `command-not-found.sh`)?

Reply:
Yes, that would be perfect.